cmd/release/cmd/tag.go

```diff
@@ -7,16 +7,29 @@ import (
 	"os"
 	"time"
 
+	"github.com/rancher/ecm-distro-tools/release/rancher"
 	"github.com/rancher/ecm-distro-tools/release/rke2"
 	"github.com/rancher/ecm-distro-tools/repository"
 	"github.com/spf13/cobra"
 )
 
+type TagRKE2CmdFlags struct {
```

Comment:
These types don't appear to be used outside of this package so I don't think they should be exported.